### ISSUE TITLE
This New Version (1.1.0) Keeps Crashing, Especially on scanning Hosts.

### ISSUE THREAD

**Issue by Kane-Kuroneko** [XITRIX/Moonlight-Switch#167]
The new version was unstable and crashed very often. Switch displayed the message  "This software has encountered an error and needs to close."

Additionally, the new version has some connection issues that become apparent during the scanning and matching of a new PC host.

https://github.com/XITRIX/Moonlight-Switch/assets/40685018/6720e294-cc64-4825-a629-10b5c3cb9487

https://github.com/XITRIX/Moonlight-Switch/assets/40685018/07aeac3e-9f08-4827-9c82-22a0a08b5ded

https://github.com/XITRIX/Moonlight-Switch/assets/40685018/867fdaf0-56ea-45f6-9b17-8fcbc95af715

**Comment by Kane-Kuroneko** [comment]
It is certainly important, as we need this message to distinguish what caused the stuttering or latency.

**Comment by VoxelTek** [comment]
Is this potentially an eMMC or SD card issue? Could you confirm if this happens with any other applications? Additionally, does physical motion (e.g. shaking the console) trigger the error? If so, chances are it doesn't relate to the application.

**Comment by VoxelTek** [comment]
Also, what Switch model are you using, and how is it hacked?

**Comment by Kane-Kuroneko** [comment]
> Is this potentially an eMMC or SD card issue? Could you confirm if this happens with any other applications? Additionally, does physical motion (e.g. shaking the console) trigger the error? If so, chances are it doesn't relate to the application.

I suppose it’s highly unlikely to be a hardware failure, as any other NRO app is working well. Additionally, once I roll back to version 1.0.0, it stopped crashing like this. Therefore, I can almost guarantee that it’s a bug in Moonlight-Switch.

> what Switch model are you using, and how is it hacked?
I am using the Switch OLED (Japanese version) and have installed the Atmosphere custom firmware. It has been hacked via a hardware chipset.

**Comment by VoxelTek** [comment]
> I suppose it’s highly unlikely to be a hardware failure, as any other NRO app is working well. Additionally, once I roll back to version 1.0.0, it stopped crashing like this. Therefore, I can almost guarantee that it’s a bug in Moonlight-Switch.

Fair enough, I just know that, when I've experienced similar issues in other applications, it has been due to intermittent connections. 
Does the issue continue to occur if the language is set to English?

> I am using the Switch OLED (Japanese version) and have installed the Atmosphere custom firmware. It has been hacked via a hardware chipset.

Most likely completely irrelevant, but what modchip type?

**Comment by XITRIX** [comment]
Is it like 100% of time crashing or it's randomly? If always crashing, it looks like SSL certificate issue, easiest way to fix just remove the content of "switch/Moonlight-Switch" folder

**Comment by XITRIX** [comment]
Try to remove the content of "switch/Moonlight-Switch" especially "key"

**Comment by Kane-Kuroneko** [comment]
> Try to remove the content of "switch/Moonlight-Switch" especially "key"尝试删除“switch/Moonlight-Switch”尤其是“key”的内容

Nothing has improved, and the issue still exists in version 1.2.1.

**Comment by XITRIX** [comment]
Should be fixed in v1.3.0
Feel free to reopen if still crashing
